examples/demos.css

```diff
@@ -29,8 +29,11 @@ body {
 .feedback {
   color: darkgrey;
   font-size: 80%;
-  height: 500px;
+  height: 350px;
```

Comment:
We could drop this completely, then it will just expand.  As it was, I almost didn't see the UA debug area below.